src/server/pfs/fuse/helpers_test.go

```diff
@@ -0,0 +1,69 @@
+package fuse
```

Comment:
can we put these helper functions in the integration test file? It doesn't look like they are being called anywhere else